aic-emu/CmdHandler.cc

```diff
@@ -26,32 +26,39 @@ using std::chrono::duration_cast;
 using std::chrono::microseconds;
 using std::chrono::system_clock;
 
-CmdHandler::CmdHandler(std::string ymlFile, std::string inputFile, AicSocketData_t* info):
-    m_ymlFileName(ymlFile),
-    m_inputFileName(inputFile),
+CmdHandler::CmdHandler(AicConfigData_t& config):
+    m_ymlFileName(config.ymlFileName),
+    m_inputFileName(config.contentFileName),
+    m_gfxDeviceStr(config.deviceString),
+    m_inputFormat(config.contentFormat),
     m_props(nullptr),
+    m_manageFps(config.manageFps),
     m_lastDispReqSentTS(0),
     m_lastDispReqYmlTS(0),
     m_firstDispReqSentTS(0)
 {
-    if (info)
+    //Check formats
+    if (m_inputFormat.compare("RGBA") != 0)
```

Comment:
I removed this for now. This is a stub field exposed for future implementation at this point: it will be re-added with necessary changes when the format is actually supported.